homeassistant/components/tesla_wall_connector/sensor.py

```diff
@@ -127,13 +127,21 @@ class WallConnectorSensorDescription(
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
     WallConnectorSensorDescription(
-        key="energy_kWh",
+        key="energy_Wh",
         name=prefix_entity_name("Energy"),
         native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
         value_fn=lambda data: data[WALLCONNECTOR_DATA_LIFETIME].energy_wh,
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    WallConnectorSensorDescription(
+        key="session_energy_Wh",
+        name=prefix_entity_name("Energy"),
+        native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
+        value_fn=lambda data: data[WALLCONNECTOR_DATA_VITALS].session_energy_wh,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
This state class is not allowed, but there's a new device class which will allow it: https://github.com/home-assistant/core/pull/88310

Either we should use the new device class, or change the state class.